### ISSUE TITLE
Bug: Convert from fbx to GLB, animation run too slowly

### ISSUE THREAD

**Issue by tungpv90** [assimp/assimp#4206]
Convert a file fbx with animation to glb, animation run too slowly
This is 2 files run in threejs

https://user-images.githubusercontent.com/22250215/143392770-0693f93d-f449-49da-955e-0b4caf637538.mp4

**Comment by philipguin** [comment]
This is almost certainly the same issue as #4197

**Comment by kimkulling** [comment]
Right, this ticket describes the same bahaviour triggered by the bug from #4197. 

Lets use #4197 to trace the issue and I will add a link to this issue here.

**Comment by tellypresence** [comment]
@tungpv90 Please try re-export of your `.fbx` to `.glb` and see if the problem still exists
